llvm/test/CodeGen/SBF/objdump_cond_op.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: llc -march=sbf -mcpu=sbfv2 -filetype=obj -o - %s | llvm-objdump -d - | FileCheck %s
+; RUN: llc -march=sbf -mcpu=sbfv2 -filetype=obj -o - %s | llvm-objdump -d --mcpu=sbfv2 - | FileCheck %s
```

Comment:
It seem a bit odd that you need to specify sbfv2 on the command line for llvm-objdump. The tool reads ELF header which contains E_FLAGS field, and that field tells whether the ELF file was generated for SBFv2. Maybe we need to fix that to work correctly?

Reply:
Good point. I'll investigate that.

Reply:
We were not reading the flags from the ELF file, so I fixed this in my last commit. The decoding of `mov32 reg, imm` is ambiguous, because it can either refer to a 32-bit register or to a 64-bit register. To circumvent this, I had to `--mattr=+alu32` in some tests.